src/test/test_rtperiodicdiff.py

```diff
@@ -1,16 +1,15 @@
 """
-Test the rtweeklydiff module
+Test the RtPeriodicDiffARProcess module
 """
 
 import jax.numpy as jnp
 import numpy as np
 import numpyro
-from jax import lax
 from jax.typing import ArrayLike
-from numpy.testing import assert_array_almost_equal, assert_array_equal
+from numpy.testing import assert_array_equal
 
 from pyrenew.deterministic import DeterministicVariable
-from pyrenew.process import RtWeeklyDiffProcess
+from pyrenew.process import RtWeeklyDiffARProcess
 
 
 def _manual_rt_weekly_diff(
```

Comment:
This doesn't get used

Reply:
Suggest a separate PR.

Reply:
Per conversation on teams, let's just remove this function?

Reply:
Removed in 7c62db6